api/src/main/java/uk/gov/hmcts/payment/api/validators/PaymentValidator.java

```diff
@@ -29,8 +36,12 @@ public void validate(Optional<String> paymentMethodType, Optional<String> servic
             dto.addFieldError("service_name", "Invalid service name requested");
         }
 
-        Optional<LocalDate> startDate = parseAndValidateDate(startDateString, "start_date", dto);
-        Optional<LocalDate> endDate = parseAndValidateDate(endDateString, "end_date", dto);
+        if ((startDateString.isPresent() && !endDateString.isPresent()) || (!startDateString.isPresent() && endDateString.isPresent())) {
```

Comment:
it's not required to have both. it should be able to work with just one parameter as well.